### DISCUSSION TITLE
Is there an updated_at value that changes for any action for a movie or TV show?

### DISCUSSION THREAD

**Discussion by sereisoglu** [trakt/trakt-api#752] [Q&A]
Hi.
The `watched_at`, `rated_at`, `hidden_at` dates in the `sync/last_activities` endpoint are updated according to the add or remove actions. But the `last_updated_at` date in movies and TV shows is not updated according to the remove action. Is there an `updated_at` value that changes for any action for a movie or TV show?

**Comment by rectifyer** [comment]
Please provide more details about the `last_updated_at` you are referring to and the specific API method you're using.

**Comment by sereisoglu** [comment]
I'm talking about the `last_updated_at` at the `sync/watched/movies` and `sync/watched/shows` endpoints.
